db/migrate/20191216113352_remove_provider_last_published_at.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveProviderLastPublishedAt < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :provider, :last_published_at
```

Comment:
I don't think the annotations ran, I still see [`last_published_at` listed in `provider.rb`](https://github.com/DFE-Digital/manage-courses-backend/pull/1060/files#diff-eeaf9199fba4aed486bc440604e87bf9R17).